.github/workflows/branch-pr-release.yaml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+    - created
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+    - run: npm install
+    - run: xvfb-run -a npm test
+      if: runner.os == 'Linux'
+    - run: npm test
+      if: runner.os != 'Linux'
+    - name: Publish
+      if: success() && startsWith( github.ref, 'refs/tags/releases/') && matrix.os == 'ubuntu-latest'
+      run: vsce publish -p ${{ secrets.VSCE_PAT }}
```

Comment:
[don't see these set yet btw](https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/settings/secrets/actions)

Reply:
yea something broke before I got there so working on that first.

Reply:
Missing some stuff from [`.travis.yml`](https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/blob/master/.travis.yml) (can remove that file in this PR as well)

https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/blob/cfd9cb5cd27bd2ca0384a6e9c42145701017e39d/.travis.yml#L14-L27

Reply:
Complete